airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.module.scss

```diff
@@ -0,0 +1,43 @@
+@use "../../../scss/colors";
+@use "../../../scss/fonts";
+
+.circle {
+  height: 20px;
+  width: fit-content;
+  min-width: 20px;
+  border-radius: 10px;
+  padding: 0 4px;
+  text-align: center;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  line-height: normal;
+
+  &.default {
+    background: colors.$dark-blue-100;
+  }
+
+  &.green {
+    background: colors.$green;
+    color: colors.$white;
```

Comment:
This needs to be `colors.$black` to have proper contrast ratio between text and background:

:heavy_check_mark: https://color.review/check/000000-67DAE1
:x: https://color.review/check/FFFFFF-67DAE1